safe-svg.php

```diff
@@ -699,6 +699,10 @@ protected function svg_dimensions( $attachment_id ) {
 		 */
 		public function disable_srcset( $image_meta, $size_array, $image_src, $attachment_id ) {
 			if ( $attachment_id && 'image/svg+xml' === get_post_mime_type( $attachment_id ) ) {
+				// Convert to array if not already to avoid PHP warnings.
+				if ( ! is_array( $image_meta ) ) {
+					$image_meta = array();
+				}
```

Comment:
I believe this will work but in thinking about this more, in a situation where somehow the `$image_meta` field isn't an array, I'm worried about the potential consequences to forcing that to be an array. In addition, in that scenario, seems like there's no need to do anything here, as the `sizes` key won't exist (since `$image_meta` isn't an array) and thus the `srcset` won't ever be created anyway.

I'd suggest instead the following change to the line above:

```php
if ( $attachment_id && 'image/svg+xml' === get_post_mime_type( $attachment_id ) && is_array( $image_meta ) ) {
```

Reply:
Agreed, it makes complete sense, thanks! (https://github.com/10up/safe-svg/pull/203/commits/7d4e730d83712655910feee3344acafe6cee8237)